### ISSUE TITLE
Assignment 1 TwoLayerNet.loss: reg_loss compute error

### ISSUE THREAD

**Issue by dc3ea9f** [bruceoutdoors/CS231n#8]
https://github.com/bruceoutdoors/CS231n/blob/c92ca904d72cd38a520e47cae557b28c9ef423ee/assignment1/cs231n/classifiers/neural_net.py#L104
Here should be
```python
    reg_loss = reg * np.sum(W1 * W1) + reg * np.sum(W2 * W2)
```
to gain the correct loss.
